From dede2a12d0bcc6ae9daf2a4671f0d618eeb44425 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Sun, 6 Mar 2016 13:56:55 +0100
Subject: [PATCH] configure.ac: Fix libavahi-client test

This only affected the report at the end of configure.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 87a9a17..2892967 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1185,7 +1185,7 @@ fi
 
 BUILD_CLIENT="no   (You need avahi-daemon and D-Bus!)"
 
-if "x$HAVE_DBUS" = "xyes" ; then
+if test "x$HAVE_DBUS" = "xyes" ; then
     BUILD_CLIENT=yes
 fi
 
